models/meshery_pattern.go

```diff
@@ -16,6 +19,26 @@ type MesheryPattern struct {
 	// but the remote provider is allowed to provide one
 	UserID *string `json:"user_id" gorm:"-"`
 
+	Location sql.Map `json:"location"`
+
 	UpdatedAt *time.Time `json:"updated_at,omitempty"`
 	CreatedAt *time.Time `json:"created_at,omitempty"`
 }
+
+// GetPatternName takes in a stringified patternfile and extracts the name from it
+func GetPatternName(stringifiedFile string) (name string, err error) {
+	out := map[string]interface{}{}
+
+	if err = yaml.Unmarshal([]byte(stringifiedFile), &out); err != nil {
+		return
```

Comment:
```suggestion
		return " ",err
```

Reply:
Aren't both the same? Or the suggestion is from code readability point of view? :thinking:

Reply:
they are same but I believe naked return is good in case of short programs not in big projects and also it better code readability(which will help other contributors) which I think we should follow. @leecalcote  any thoughts?
Thanks

Reply:
Ah, I see. On the contrary, I believe that the functions which are public facing, (i.e are not in `internal` directory or not being exported) and are short in their size (so that chances of shadowing are minimal) then named returns helps in "self-documenting". 

However, I do agree that this maybe less readable in some senses :thinking:

Reply:
Updated the PR. Replaced named and naked returns with explicit returns. Thanks for reviewing @subhamkrai! :smile:

Reply:
@utkarsh-pro thanks😀